pkg/config/system_probe.go

```diff
@@ -172,6 +172,8 @@ func InitSystemProbeConfig(cfg Config) {
 
 	cfg.BindEnvAndSetDefault(join(smNS, "enable_java_tls_support"), false)
 	cfg.BindEnvAndSetDefault(join(smNS, "java_agent_args"), defaultServiceMonitoringJavaAgentArgs)
+	cfg.BindEnvAndSetDefault(join(smNS, "java_agent_allow_regex"), "")
+	cfg.BindEnvAndSetDefault(join(smNS, "java_agent_block_regex"), "")
```

Comment:
Is it worth adding another level for Java? for example:
```
service_monitoring_config:
  java_tls:
    enabled: true
    args:
    allow_regex:
    block_regex:
```

Reply:
to be consistent enable on other protocols/binary are
```
enable_https_monitoring
...
enable_go_tls_support
enable_java_tls_support

```
but we can add the option under `java_agent` key
@guyarb WDYT ?

Reply:
I changed the format for the optional arguments

Reply:
why?
as a customer it make less sense to have different locations for the configuration
less intuitive, and more error prune

Reply:
done